### PR TITLE
Add "Introducing the Private Hub" blog post

### PR DESCRIPTION
**Pull request by feconroses**
## What does this PR do?
Adds a new blog post to introduce the Private Hub, why it’s useful, and how companies can leverage it to accelerate their ML roadmaps.

Jeff, Britney, Rajiv, and Julien Simon kindly [provided feedback on the draft on Google Docs](https://docs.google.com/document/d/1Q8VQ_zt6luwuOd703aQDH7YT8RplOpFCZWErxHZDi9Q/edit?usp=sharing) (most comments/suggestions are closed now).

The goal is to publish this blog post on Aug 3 as on Aug 5 I start my vacations.

Relevant Slack thread: https://huggingface.slack.com/archives/C0259GJV3EJ/p1658870902335809 

Note: we recently decided to [rename "Private Model Hub" to "Private Hub"](https://huggingface.slack.com/archives/C0235V2JNLV/p1658948441253829).

## Screenshot
ph-1
ph-2

### CODE REVIEW

**Comment by feconroses** [comment]
Added an awesome custom thumbnail created by @Reeray for this post.

File: introducing-private-hub.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+<figure class="image table text-center m-0 w-full">
+  <medium-zoom background="rgba(0,0,0,.7)" alt="Super Glue dataset preview" src="assets/92_introducing_private_hub/dataset-preview.png"></medium-zoom>
+  <figcaption>Super Glue dataset preview</figcaption>
+</figure>
+
+### Spaces
+
+A few months ago, we introduced a new feature on the 🤗 Hub called [Spaces](https://huggingface.co/spaces/launch). It's a simple way to build and host machine learning apps. Spaces allow you to easily showcase your ML models to business stakeholders and get the feedback you need to move your ML project forward. 
+
+If you've been generating funny images with [DALL-E mini](https://huggingface.co/spaces/dalle-mini/dalle-mini), then you have used Spaces. This space showcase the [DALL-E mini model](https://huggingface.co/dalle-mini/dalle-mini), a machine learning model to generate images based on text prompts:
```
Comment (osanseviero) [inline introducing-private-hub.md:129]: nit: sometimes you use spaces and sometimes Spaces

File: introducing-private-hub.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+### Training accurate models faster
+
+#### Leveraging a pre-trained model from the Hub
+
+Instead of training models from scratch, transfer learning now allows you to build more accurate models 10x faster ⚡️by fine-tuning pre-trained models available on the Hub for your particular use case. 
+
+For our demo example, one of the requirements for building this ML app for financial analysts is doing sentiment analysis. Business stakeholders want to automatically get a sense of a company's performance as soon as financial docs and analyst reports are available.
+
+So as a first step towards creating this ML app, we dive into the [🤗 Hub](https://huggingface.co/models) and explore what pre-trained models are available that we can fine-tune for sentiment analysis. The search bar and tags will let us filter and discover relevant models very quickly. Soon enough, we come across [FinBERT](https://huggingface.co/yiyanghkust/finbert-pretrain), a BERT model pre-trained on corporate reports, earnings call transcripts and financial analyst reports:
```
Comment (osanseviero) [inline introducing-private-hub.md:192]:
```suggestion
So as a first step towards creating this ML app, we dive into the [Hub](https://huggingface.co/models) and explore what pre-trained models are available that we can fine-tune for sentiment analysis. The search bar and tags will let us filter and discover relevant models very quickly. Soon enough, we come across [FinBERT](https://huggingface.co/yiyanghkust/finbert-pretrain), a BERT model pre-trained on corporate reports, earnings call transcripts and financial analyst reports: 
```

File: introducing-private-hub.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+  <figcaption>Testing the fine-tuned models with the Inference Widget</figcaption>
+</figure>
+
+### Easily demo models to relevant stakeholders
+
+Now that we have trained our custom model for analyzing financial documents, as a next step, we want to build a machine learning demo with [Spaces](https://huggingface.co/spaces/launch) to validate our MVP with our business stakeholders. This demo app will use our custom sentiment analysis model, as well as a second FinBERT model we fine-tuned for [detecting forward-looking statements](https://huggingface.co/FinanceInc/finbert_fls) from financial reports. This interactive demo app will allow us to get feedback sooner, iterate faster, and improve the models so we can use them in production. ✅
+
+In less than 20 minutes, we were able to build an [interactive demo app](https://huggingface.co/spaces/FinanceInc/Financial_Analyst_AI) that any business stakeholder can easily test right from their browsers:
+
+<figure class="image table text-center m-0 w-full">
```
Comment (osanseviero) [inline introducing-private-hub.md:241]: Consider embedding the space in this demo as per https://gradio.app/sharing_your_app/#embedding-hosted-spaces

**Review by osanseviero — APPROVED** [review]
Very cool! I only left nits, but this is a very nice read! I also opened some PRs to the space which hopefully help a bit :) https://huggingface.co/spaces/FinanceInc/Financial_Analyst_AI/discussions

File: introducing-private-hub.md [hunk 4]
```diff
@@ -0,0 +1,297 @@
+---
+title: "Introducing the Private Hub: A New Way to Build With Machine Learning"
+thumbnail: /blog/assets/92_introducing_private_hub/thumbnail.png
+---
+
```
[the remark this answers is not in the document]
Comment (feconroses) [reply introducing-private-hub.md:5]: Thanks for pointing it out, I totally forgot to compress them. Fixed this :)

File: introducing-private-hub.md (hunk 1 above)
Comment (feconroses) [reply introducing-private-hub.md:129]: I see Spaces as the feature and spaces as the asset owned/created by the users (like in "_a user that models, datasets and spaces_").

File: introducing-private-hub.md (hunk 2 above)
Comment (feconroses) [reply introducing-private-hub.md:192]: I think it's better to keep the space.

File: introducing-private-hub.md (hunk 3 above)
Comment (feconroses) [reply introducing-private-hub.md:241]: I think I prefer not to embed it as I don't want to draw too much attention to the demo; I would prefer if most readers just keep reading the post but provide the link to give the possibility to those interested in checking out.

File: introducing-private-hub.md [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+</figure>
+
+The 🤗 Hub is also a central place for feedback and development in machine learning. Teams use [pull requests and discussions](https://huggingface.co/docs/hub/repositories-pull-requests-discussions) to support peer reviews on models, datasets, and spaces, improve collaboration and accelerate their ML work.
+
+<figure class="image table text-center m-0 w-full">
+  <medium-zoom background="rgba(0,0,0,.7)" alt="Pull requests and discussions on a model" src="assets/92_introducing_private_hub/pull-requests-and-discussions.png"></medium-zoom>
+  <figcaption>Pull requests and discussions on a model</figcaption>
+</figure>
+
+The Hub allows users to create [Organizations](https://huggingface.co/docs/hub/organizations), that is, team accounts to manage models, datasets, and spaces collaboratively. An organization’s repositories will be featured on the organization’s page and admins can set roles to control access to these repositories. Every member of the organization can contribute to models, datasets and spaces given the right permissions. Here at Hugging Face, we believe having the right tools to collaborate drastically accelerates machine learning development! 🔥
```
[the remark this answers is not in the document]
Comment (feconroses) [reply introducing-private-hub.md:71]: Good idea! Added a link to BigScience org!

File: introducing-private-hub.md [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+
+<figure class="image table text-center m-0 w-full">
+  <medium-zoom background="rgba(0,0,0,.7)" alt="Datasets available for sentiment classification" src="assets/92_introducing_private_hub/filtering-datasets.png"></medium-zoom>
+  <figcaption>Datasets available for sentiment classification</figcaption>
+</figure>
+
+Similar to models, datasets uploaded to the 🤗 Hub have [Dataset Cards](https://huggingface.co/docs/hub/datasets-cards#dataset-cards) to help users understand the contents of the dataset, how the dataset should be used, how it was created and know relevant considerations for using the dataset. You can use the [Dataset Viewer](https://huggingface.co/docs/hub/datasets-viewer) to easily view the data and quickly understand if a particular dataset is useful for your machine learning project:
+
```
[the remark this answers is not in the document]
Comment (feconroses) [reply introducing-private-hub.md:119]: Thanks for the suggestion but I prefer not to, so we can keep it simple.

File: introducing-private-hub.md [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+<figure class="image table text-center m-0 w-full">
+  <medium-zoom background="rgba(0,0,0,.7)" alt="Datasets available for sentiment classification" src="assets/92_introducing_private_hub/filtering-datasets.png"></medium-zoom>
+  <figcaption>Datasets available for sentiment classification</figcaption>
+</figure>
+
+Similar to models, datasets uploaded to the 🤗 Hub have [Dataset Cards](https://huggingface.co/docs/hub/datasets-cards#dataset-cards) to help users understand the contents of the dataset, how the dataset should be used, how it was created and know relevant considerations for using the dataset. You can use the [Dataset Viewer](https://huggingface.co/docs/hub/datasets-viewer) to easily view the data and quickly understand if a particular dataset is useful for your machine learning project:
```
[the remark this answers is not in the document]
Comment (feconroses) [reply introducing-private-hub.md:118]: I think it's better to keep the space.

**Comment by feconroses** [comment]
@osanseviero thanks so much for taking the time to share your feedback and suggestions. Just made a new commit with various fixes and improvements.

**Pull request merged**